frontend/__tests__/unit/pages/About.test.tsx

```diff
@@ -35,6 +35,78 @@ jest.mock('utils/aboutData', () => ({
     'This is a test paragraph about the project.',
     'This is another paragraph about the project history.',
   ],
+  leaders: {
+    arkid15r: 'CCSP, CISSP, CSSLP',
+    kasya: 'CC',
+    mamicidal: 'CISSP',
+  },
+  missionContent: {
+    mission:
+      'OWASP Nest is a comprehensive platform designed to empower developers, security professionals, and organizations in their journey toward building secure applications.',
+    whoItsFor:
+      'OWASP Nest is designed for developers, security professionals, project leaders, and organizations looking to integrate security best practices into their development workflows.',
+  },
+  keyFeatures: [
+    {
+      title: 'Advanced Search Capabilities',
+      description:
+        'Find OWASP projects, tools, and resources with precision using our intelligent search functionality.',
+    },
+    {
+      title: 'Slack Integration',
+      description: 'Stay connected with the OWASP community through seamless Slack integration.',
+    },
+    {
+      title: 'OWASP Chapters Proximity Page',
+      description:
+        'Discover local OWASP chapters and events near you to engage with the community.',
+    },
+    {
+      title: 'AI-Generated Insights',
+      description:
+        'Leverage AI-powered insights to better understand project relationships and recommendations.',
+    },
+  ],
+  getInvolvedContent: {
```

Comment:
Was it intended to look like this?

![Screenshot 2025-06-16 at 8 38 07 PM](https://github.com/user-attachments/assets/fca07b7a-d04d-4a68-923f-d56b089612f3)

The list item markers are misaligned and the links color isn't the one we normally use.